test/e2e/suite_test.go

```diff
@@ -107,6 +107,7 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 	testNamespace = fmt.Sprintf("karmadatest-%s", rand.String(RandomStrLength))
 	err = setupTestNamespace(testNamespace, kubeClient)
+	framework.WaitNamespacePresentOnClusters(framework.ClusterNames(), testNamespace)
```

Comment:
We can move this line down one line. The following line is used to check whether the namespace is successfully created.